include/motor-control/core/tasks/gear_tmc2160_motor_driver_task.hpp

```diff
@@ -82,11 +82,23 @@ class MotorDriverMessageHandler {
         LOG("Received read motor driver request: addr=%d", m.reg_address);
         uint32_t data = 0;
         if (tmc2160::registers::is_valid_address(m.reg_address)) {
-            driver.read(tmc2160::registers::Registers(m.reg_address), data,
-                        m.message_index);
+            auto converted_addr = static_cast<uint8_t>(m.reg_address);
+            uint32_t token = spi::utils::build_token(converted_addr);
+            driver.read(token, data, m.message_index);
         }
     }
 
+    void handle(const can::messages::ReadMotorDriverErrorStatus& m) {
+        LOG("Received read motor driver error register request");
+        uint32_t data = 0;
+        auto converted_addr =
+            static_cast<uint8_t>(tmc2160::registers::Registers::DRVSTATUS);
+        std::array tags{spi::utils::ResponseTag::IS_ERROR_RESPONSE};
+        uint32_t token = spi::utils::build_token(
+            converted_addr, spi::utils::byte_from_tags(tags));
+        driver.read(token, data, m.message_index);
```

Comment:
here, for instance, let's make `driver.read()` take a byte of tags or something in addition to the register address